easybuild/easyconfigs/a/AlphaPulldown/AlphaPulldown-0.30.4_fix-deps.patch

```diff
@@ -0,0 +1,34 @@
+# making AlphaPulldown's dependencies less strict
```

Comment:
some of these are making it a lot less strict. Especially TF from 2.9 to 2.4. Is this known to still work correctly?

Reply:
well, we need the `2020b` toolchain, as the `cctbx-base` binary is built for that and there are no alternatives...
when you brought it up, I actually checked the usage and differences between versions.
I've found that all the usages of `tensorflow` appear to begin with `import tensorflow.compat.v1`, which suggests that there is no reason to keep the deps this strict.

Reply:
I looked into this with Denis, and unfortunately we indeed are forced to stick to Python 3.8 here, unless *a lot* more effort is made to figure out how to get a from-source install of `cctbx-base` to work (which doesn't seem trivial).

So we're proceeding under the assumption that the version requirements here are a bit too strict.
If problems emerge when using `AlphaPullDown` because of older dependency versions, we'll revisit this then